config/base.yaml

```diff
@@ -8,6 +8,7 @@ api_key:
   - <SECURITY_SERVER_ROLE_NAME>
   - <SECURITY_SERVER_ROLE_NAME>
   - <SECURITY_SERVER_ROLE_NAME>
+  - <SECURITY_SERVER_ROLE_NAME>
```

Comment:
This section is getting sillier as the roles are added, how about just adding them all here and be done with them or avoiding the repetition a'la:

```
-  <SECURITY_SERVER_ROLE_NAME>
-  # ...  <SECURITY_SERVER_ROLE_NAME> ...
```